### PR TITLE
fix lint errors.

### PR DESCRIPTION
**Pull request by ingarabr**

### CODE REVIEW

**Review by steinaol — APPROVED** [review]
LGTM

File: src/components/layout/Breadcrumb.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ import map from 'lodash/map';
 import { OverlayTrigger, Popover } from 'react-bootstrap';
 
 const tooltip = (tipText, child) =>
-  <OverlayTrigger placement="top" overlay={<Popover id={"breadcrumb tooltip"}>{tipText}</Popover>}>
+  <OverlayTrigger placement="top" overlay={<Popover id={'breadcrumb tooltip'}>{tipText}</Popover>}>
```
Comment (jarlah) [inline src/components/layout/Breadcrumb.js]: actually, you can remove the curlies

**Review by jarlah — CHANGES_REQUESTED** [review]

File: src/components/layout/Breadcrumb.js (hunk 1 above)
Comment (jarlah) [reply src/components/layout/Breadcrumb.js]: and revert back to double quotes

Comment (ingarabr) [reply src/components/layout/Breadcrumb.js]: ah, I see :)

**Review by jarlah — APPROVED** [review]

**Pull request merged**